llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp

```diff
@@ -11379,7 +11379,7 @@ InstructionCost BoUpSLP::getSpillCost() const {
           if (auto *FPMO = dyn_cast<FPMathOperator>(II))
             FMF = FPMO->getFastMathFlags();
           IntrinsicCostAttributes ICA(II->getIntrinsicID(), II->getType(), Tys,
-                                      FMF);
+                                      FMF, II);
```

Comment:
Unrelated change?

Reply:
![image](https://github.com/user-attachments/assets/feb6f54f-8f14-41f8-80fe-115ac51f829a)
vp_icmp and vp_fcmp nend to get Inst.